server/helpers/procBuilder.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package server
+package helpers
```

Comment:
Most part of this package seem to be more than "just" helpers to me 🤔

Reply:
Yeah. They're used by both `server/api` and `server/grpc`. Some provide logic around updating models. Others have core logic, but they don't directly serve api or grpc, so I wanted to put them somewhere else to make navigating the API code more straight forward.

Do you have any suggestions for a better name or a better way to organize?

Reply:
K. I moved `server/helpers` to `server/shared`